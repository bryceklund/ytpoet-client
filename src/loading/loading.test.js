import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'
import Loading from './loading'

describe('Loading', () => {
it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter>
                    <Loading loading={true} />
                    </BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
})
it('matches its snapshot', () => {
    const tree = renderer.create(<BrowserRouter>
                                <Loading loading={true} />
                                </BrowserRouter>)    
                                .toJSON()
    expect(tree).toMatchSnapshot()
})
})