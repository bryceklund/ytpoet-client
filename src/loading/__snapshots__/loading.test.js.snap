// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading matches its snapshot 1`] = `
<div
  className="loading"
  id="loading"
/>
`;
